bot/menu.py

```diff
@@ -78,7 +86,7 @@ def add_menu(self, tags):
         tags : Sequence[ForumTag]
             The tags owned by the Forum channel
         """
-        menu = Menu(tags)
+        menu = Menu(tags, self.sub_toggle)

```

Comment:
Does the menu not allow us to add/remove within the menu like a series of available tags and a switch to turn them on/off? 

I dont recall what the menu looks like right now, got any screenshots as well?

Reply:
![image](https://github.com/BobaTalks/bobabot/assets/116035879/910406d3-310e-4048-8edf-857eccdd7b25)
The way this currently works is like an on/off switch for the callback in the menu. It allows us to use the same menu object and pass in a simple set of tags but just to specify whether we want to add or remove subscribers.